godot-core/src/engine/script_instance.rs

```diff
@@ -243,7 +173,10 @@ impl<T: ScriptInstance> Drop for ScriptInstanceData<T> {
 ///
 /// The exact GDExtension type of the pointer is `sys::GDExtensionScriptInstancePtr`, but you can treat it like an opaque pointer.
 #[must_use]
-pub fn create_script_instance<T: ScriptInstance>(rs_instance: T) -> *mut c_void {
+pub fn create_script_instance<T: ScriptInstance>(
+    rs_instance: T,
```

Comment:
While we're at it, please rename this to `rust_instance` :slightly_smiling_face: